tests/metrics/functional/test_classification.py

```diff
@@ -294,3 +297,28 @@ def test_dice_score():
 
 # example data taken from
 # https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/tests/test_ranking.py
+
+
+def test_iou():
+    # Generate dataset
+    target = (torch.arange(120) % 3).view(-1, 1)
```

Comment:
can you also use pytest parametrization? That way we can easily add additional testcases :)

Reply:
Updated